packages/terra-time-input/src/TimeInput.module.scss

```diff
@@ -68,6 +70,12 @@
     }
   }
 
+  .time-input-hour[disabled],
+  .time-input-minute[disabled],
+  .meridiem-display[disabled] {
+    opacity: 1;
+  }
+
```

Comment:
@amichaelparker - this actually doesn't work without it chained & nested below the .time-input.disabled classes, otherwise we will need to refactor. For the mobile view [see twelve-hour/mobile test](https://terra-core-deployed-pr-1794.herokuapp.com/#/tests/terra-time-input/time-input/twelve-hour/mobile) and reduce your screen width below 1024px, you'll see this is when we want the original opacity to not be overridden.